### PR TITLE
Jk/ps notification

### PR DESCRIPTION
**Pull request by j-kanuch**
## Description of changes/additions
This PR implements a tool to manage Pub/Sub notifications. The tool allows listing, creating, and deleting pub/sub notifications on a Google Cloud Storage bucket. Listing of pub/sub notifications can be filtered on bucket or notification id.
The JSON configuration files located at `rdr_service/config/pub_sub_config_[env].json` contain the configurations for the pub-sub notifications.
Usage documentation is located here: [Playbook](https://docs.google.com/document/d/1FkzFyc_neXn4gAQfdaCoe91mogD4frIAOFtXDeDM-sU/edit?ts=5d6fc4db#).

## Tests
Tested on sandbox.

### CODE REVIEW

File: rdr_service/tools/tool_libs/pubsub_notification_manager.py [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    def delete_notification_from_config(self):
+        config_path = self.get_config_path()
+
+        with open(config_path, "r") as f:
+            config_data = json.load(f)
+
+        notification_iter = filter(
+            lambda x: x['id'] == str(self.args.id),
+            config_data['notifications']
+        )
```
Comment (dtharpeuno) [inline rdr_service/tools/tool_libs/pubsub_notification_manager.py:207]: Nice

**Review by dtharpeuno — APPROVED** [review]
Nice job! If the config is updated when running this locally, we will need to commit the updated config file right?

**Review by kenny-skaggs — APPROVED** [review]
This will be really helpful :smile:

**Comment by j-kanuch** [comment]
> Nice job! If the config is updated when running this locally, we will need to commit the updated config file right?

Yes, the updated config file will need to be committed.

**Pull request merged**